### ISSUE TITLE
'Server cannot be contacted' error during installation

### ISSUE THREAD

**Issue by Sinemora10** [gnodipac886/MatebookXPro-hackintosh#35]
Hi! I've managed to prepare a working installation usb with Mojave 10.14.6 using the script 'gibMacOS' referenced to in another post. Everything seems fine until I start the actual installation when I am met with a "recovery server could not be contacted" error. Anyone experienced it before? What can I do?

**Comment by ayushyamitabh** [comment]
Not sure if you still need help with this but, I solved it by using a USB-C to Ethernet adapter and including Realtek1811 or IntelMausi Kext (can't remember which one) you can put both in your kexts folder shouldn't be a problem

**Comment by Sinemora10** [comment]
Hi Auysh, thank you very much for taking the time to reply my query. Yes, I sort of moved on from the Catalina / Mojave hackintosh project, but mainly because after playing with both OSs on a friend's MB (so, in theory, having the original 'Apple' experience as it is intended) I decided that neither is worth going through the hassle of installing on the Matebook. At least not when there's a perfectly working, clean and even more elegant alternative in the Linux world (I currently am on Ubuntu 19.04 with the 'old' Unity7 DE). 

That said, I would consider investing time in an hackintosh project if I could install Snow Leopard on the Matebook X Pro. I know this is OT but I would very much appreciate any information or help in this regard (I tried scouring the internet for guidance but so far I found very little). If such project is even feasible, that is.

Thanks again for the courtesy of replying.

LSdC
